salt/srv/salt/base/calamari-crush-location.py

```diff
@@ -20,14 +20,14 @@ def get_last_crush_location(osd_id):
     key = 'daemon-private/osd.%s/v1/calamari/osd_crush_location' % osd_id
     osd_keyring = '/var/lib/ceph/osd/ceph-%s/keyring' % osd_id
     admin_keyring = '/etc/ceph/ceph.client.admin.keyring'
-    commands = (['sudo', 'ceph', '--name', 'osd.%s' % osd_id, '--keyring', osd_keyring, 'config-key', 'get', key],
+    commands = (['ceph', '--name', 'osd.%s' % osd_id, '--keyring', osd_keyring, 'config-key', 'get', key],
```

Comment:
Unrelated to the change, but looking at this again I notice the hardcoded "/etc/ceph/ceph.client.admin.keyring" -- on a well configured system the OSDs may well not have an admin keyring -- that would be another source of failures here.